client/me/get-apps/index.jsx

```diff
@@ -13,78 +13,22 @@ import Main from 'components/main';
 import Button from 'components/button';
 import Card from 'components/card';
 import SectionHeader from 'components/section-header';
+import Illustration from 'components/illustration';
 
 export const GetApps = ( { translate } ) => (
 	<Main className="get-apps">
 		<MeSidebarNavigation />
-		<SectionHeader label={ translate( 'Desktop apps' ) } />
+		<Illustration illustration={ 'empty-results' } />
+		<SectionHeader label={ translate( 'WordPress Apps' ) } />
 		<Card className="get-apps__desktop">
-
-			<section className="get-apps__app">
-				<div className="get-apps__app-icon">
-					<img src="/calypso/images/me/get-apps-osx.svg" alt={ translate( 'Mac' ) } width="96" height="96" />
-				</div>
-				<p><strong>{ translate( 'Mac' ) }</strong><br />
-				{ translate( 'Requires 10.11 or newer' ) }</p>
-				<Button href="https://apps.wordpress.com/d/osx?ref=getapps">{ translate( 'Download', { context: 'verb' } ) }</Button>
-			</section>
-
-			<section className="get-apps__app">
-				<div className="get-apps__app-icon">
-					<img src="/calypso/images/me/get-apps-windows.svg" alt={ translate( 'Windows' ) } width="96" height="96" />
-				</div>
-				<p><strong>{ translate( 'Windows' ) }</strong><br />
-				{ translate( 'Requires 7 or newer' ) }</p>
-				<Button href="https://apps.wordpress.com/d/windows?ref=getapps">{ translate( 'Download' ) }</Button>
-			</section>
-
-			<section className="get-apps__app">
-				<div className="get-apps__app-icon">
-					<img src="/calypso/images/me/get-apps-linux.svg" alt={ translate( 'Linux' ) } width="96" height="96" />
-				</div>
-				<p><strong>{ translate( 'Linux' ) }</strong><br />
-				{ translate( 'Choose your distribution' ) }</p>
-				<Button href="https://apps.wordpress.com/d/linux?ref=getapps">
-					{ translate( '.TAR.GZ' ) }
-				</Button>
-				<Button href="https://apps.wordpress.com/d/linux-deb?ref=getapps">
-					{ translate( '.DEB' ) }
-				</Button>
-			</section>
-
+			<p>{ translate( 'Desktop App for Mac' ) }</p>
```

Comment:
Hi! I've found a possible matching string that has already been translated **7** times:
`translate( 'Desktop App' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).